code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/ConsentExtension.java

```diff
@@ -174,13 +177,24 @@ void handleConsentUpdate(@NonNull final Event event) {
 			return;
 		}
 
+		final boolean outsideTimeout =
```

Comment:
nit: was the `canSync` function meant to be called here rather than duplicating the code within?